modules/xhbBidAdapter.js

```diff
@@ -1,44 +1,42 @@
-const CONSTANTS = require('src/constants.json');
-const utils = require('src/utils.js');
-const adloader = require('src/adloader.js');
-const bidmanager = require('src/bidmanager.js');
-const bidfactory = require('src/bidfactory.js');
-const adaptermanager = require('src/adaptermanager');
-
-function XhbAdapter() {
-  const _defaultBidderSettings = {
-    alwaysUseBid: true,
-    adserverTargeting: [
-      {
-        key: 'hb_xhb_deal',
-        val: function (bidResponse) {
-          return bidResponse.dealId;
-        }
-      },
-      {
-        key: 'hb_xhb_adid',
-        val: function (bidResponse) {
-          return bidResponse.adId;
-        }
-      }
-    ]
+import * as Adapter from 'src/adapter';
+import bidfactory from 'src/bidfactory';
+import bidmanager from 'src/bidmanager';
+import * as utils from 'src/utils';
+import { STATUS } from 'src/constants';
+import adaptermanager from 'src/adaptermanager';
+import { loadScript } from 'src/adloader';
+
+const XhbAdapter = function XhbAdapter() {
+  const baseAdapter = Adapter.createNew('xhb');
+  let usersync = false;
+
+  baseAdapter.callBids = function (params) {
+    const anArr = params.bids;
+    for (let i = 0; i < anArr.length; i++) {
+      let bidRequest = anArr[i];
+      let callbackId = bidRequest.bidId;
+      loadScript(buildJPTCall(bidRequest, callbackId));
+    }
   };
-  bidmanager.registerDefaultBidderSetting('xhb', _defaultBidderSettings);
```

Comment:
It was originally, yes - but on second though, it probably is better to keep it in for the time being, so I added it in again.